src/components/organisms/ReplayCardView/ReplayCardView.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import tempImg from '@assets/images/box_c_three.png';
+import A from '@atoms';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  border-radius: 10px;
+  width: 334px;
+  height: 368px;
+  box-shadow: 0 0.6vh 1.2vh 0 rgba(4, 4, 161, 0.1);
```

Comment:
통일성을 위해 기준을 vh로 변경해주시면 좋을 것 같습니다. 기준은 현재 px 단위에 10을 나눈 값을 주면 됩니다.

추후에 px이나 rem 단위로 리펙토링을 다시하게 될 듯 합니다.

```suggestion
  display: flex;
  flex-direction: column;
  border-radius: 1vh;
  width: 33.4vh;
  height: 36.8vh;
  box-shadow: 0 0.6vh 1.2vh 0 rgba(4, 4, 161, 0.1);
```